packages/preview-middleware-client/src/cpe/outline/nodes.ts

```diff
@@ -136,15 +142,40 @@ export async function transformNodes(input: OutlineViewNode[], scenario: Scenari
     return items;
 }
 
+/**
+ * Fill reuse components ids.
+ *
+ * @param reuseComponentsIds ids of reuse components that are filled when outline nodes are transformed
+ * @param node view node
+ * @param scenario type of project
+ */
+async function fillReuseComponents(
+    reuseComponentsIds: Set<string>,
+    node: OutlineViewNode,
+    scenario: Scenario
+): Promise<void> {
+    const version = sap.ui.version;
+    const minor = parseInt(version.split('.')[1], 10);
+    if (scenario === 'ADAPTATION_PROJECT' && node?.component && minor >= 114) {
```

Comment:
This logic looks quite similar to the `isReuseComponent` function in `init-dialogs.ts`. We should have a common function for this in `cpe/utils.ts` and export it from the `cpe` and then it can be reused in `adp` similar to the `init` function.

Reply:
fixed in #[60db612](https://github.com/SAP/open-ux-tools/pull/1941/commits/60db612ddc2c534978b7d71f074f12bbba1da920)